ecosystem/sep-0024.md

```diff
@@ -83,11 +83,12 @@ SEP-24 lays out many options for how deposit and withdrawal can work. These are
     * If the `/fee` endpoint is enabled, use it for computing fees when you need to show them to the user.
 * **Authentication**
    * Perform [authentication](#authentication) via SEP-10 before hitting those endpoints
+* **Requested Fields**
+   * If the anchor requests personal information in `/info`, and your app has it or wishes to collect it, send the information to the anchor via [SEP-12](sep-0012.md).
```

Comment:
SEP-12 is currently deprecated. We need to un-deprecate it if we're going to recommend its use.